src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  let total = days * dayPrice;
+
+  if (days >= 3 && days < 7) {
+    total = total - 20;
+  } else if (days >= 7) {
+    total = total - 50;
+  }
```

Comment:
The use of literals in expressions is discouraged. Instead, define constants that explain what these values represent, such as `MIN_DAYS_FOR_DISCOUNT` and `LONG_TERM_DISCOUNT`. Refer to checklist item 2 for an example of how to implement constants for better readability and maintainability.

Reply:
The variables names used by me are readable and brief. I can change them into 'DAY_PRICE' and 'TOTAL_PRICE' if a human mentor says so but I'm not sure if it's needed in a such simple function with only 3 variables.